group-coordinator/src/main/java/org/apache/kafka/coordinator/group/generic/GenericGroup.java

```diff
@@ -823,9 +823,17 @@ public void validateOffsetCommit(
 
     /**
      * Validates the OffsetFetch request.
+     *
+     * @param memberId              The member id. This is not provided for generic groups.
+     * @param memberEpoch           The member epoch for consumer groups. This is not provided for generic groups.
+     * @param lastCommittedOffset   The last committed offsets in the timeline.
      */
     @Override
-    public void validateOffsetFetch() throws GroupIdNotFoundException {
+    public void validateOffsetFetch(
+        String memberId,
```

Comment:
For my understanding, we pass these to be consistent with the interface, but we don't need it?

Reply:
Correct. I mentioned this in the javadoc of the method.